### ISSUE TITLE
ValueError in sphinx/util/compat.py from docutils pre-release version "0.14rc1"

### ISSUE THREAD

**Issue by glennmatthews** [sphinx-doc/sphinx#3838]
Subject: ValueError in sphinx/util/compat.py from docutils pre-release version "0.14rc1"

### Problem
- When pre-release `docutils==0.14rc1` is installed, `import sphinx.util.compat` throws a `ValueError`.

#### Procedure to reproduce the problem
```
pip install docutils==0.14rc1
pip install sphinx
python
>>> import sphinx.util.compat
```

#### Error logs / results
```
>>> import sphinx.util.compat
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "py36-test/lib/python3.6/site-packages/sphinx/util/compat.py", line 21, in <module>
    docutils_version = tuple(int(x) for x in _du_version.split('.')[:2])
  File "py36-test/lib/python3.6/site-packages/sphinx/util/compat.py", line 21, in <genexpr>
    docutils_version = tuple(int(x) for x in _du_version.split('.')[:2])
ValueError: invalid literal for int() with base 10: '14rc1'
```

#### Expected results
Not to throw an error, especially in response to a simple `import`. :-)

### Reproducible project / your project
- Trivially recreatable in a virtualenv.

### Environment info
- OS: Mac, Ubuntu
- Python version: reproduced on 3.6
- Sphinx version: 1.4.9, 1.6.2

**Comment by jfbu** [comment]
Hi, thanks and sorry for inconvenience. This is #3821 and was fixed at ff1adc7. Please wait Sphinx 1.6.3.

Notice that sphinx.util.compat is deprecated and will be removed at Sphinx 1.7.

refs: http://www.sphinx-doc.org/en/master/changes.html#features-removed

**Comment by glennmatthews** [comment]
Thanks for the information!
